src/js/pages/attendee/Home.jsx

```diff
@@ -149,19 +147,23 @@ export default class Featured extends React.Component {
         right++;
       }
     }
-    this.setState({sponsors: sponsorsSplit});
+    this.setState({ sponsors: sponsorsSplit });
   }
 
   render() {
-    const { messages, events, notifications, logo, branding, locations, cards, contacts, surveys, sponsors, preferences} = this.state;
+    const { messages, events, notifications, logo, branding, locations, cards, contacts, surveys, sponsors, preferences } = this.state;
 
-    var eventCount = 0;
+    let eventCount = 0;
+    const now = moment();
     const EventComponents = events.map((event) => {
-      if (moment().diff(moment(event.starttime), "hours") < 2 || eventCount < 3) {
+      const hourDiff = moment(event.starttime).diff(now, 'hours');
+      if ((hourDiff >= -1 && hourDiff <= 2) || eventCount < 3) {
+
         eventCount++;
-        return <Card type="event" key={"event-" + event.id} item={event}/>;
+        return <Card type="event" key={`event-${event.id}`} item={event} />;
       }
     });
+    EventComponents.sort((a, b) => a.props.item.starttime - b.props.item.starttime);
```

Comment:
we might want to use `moment(a.props.item.starttime).isBefore` to stick with the moment stuff. Nothing wrong with this approach either.